pkgs/by-name/fa/fastd-exporter/package.nix

```
@@ -0,0 +1,35 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+
+buildGoModule {
+  pname = "fastd-exporter";
+  version = "0-unstable-2024-04-09";
+
+  src = fetchFromGitHub {
+    owner = "freifunk-darmstadt";
+    repo = "fastd-exporter";
+    rev = "374e4334af6661f4c91a3e83bf7ce071a2a72eca";
+    sha256 = "sha256-0oU4+9G19XP5qtGdcfMz08T04hjcoXX/O+FkaUPxzXE=";
```

`sha256 =` is deprecated, IIRC, so:


```suggestion
    hash = "sha256-0oU4+9G19XP5qtGdcfMz08T04hjcoXX/O+FkaUPxzXE=";
```